.gitignore

```diff
@@ -10,3 +10,6 @@ sandbox.db
 .coverage
 
 coverage/
+
+# TODO remove this once migrations are shipped with the code
+*/migrations/*
```

Comment:
We need to remove this since migrations are being added from production. Make sure to regenerated before merging feature branch.